app/frontend/src/components/HorizontalScroller.tsx

```diff
@@ -0,0 +1,56 @@
+import { Box, makeStyles } from "@material-ui/core";
+import classNames from "classnames";
+import React, { ReactNode } from "react";
+
+import useOnVisibleEffect from "../utils/useOnVisibleEffect";
+import CircularProgress from "./CircularProgress";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex",
+    flexDirection: "row",
+    alignItems: "center",
+    overflowX: "auto",
+    WebkitOverflowScrolling: "touch",
+    padding: theme.spacing(2),
+    "& > *": {
+      flex: "0 0 auto",
+    },
+  },
+  loader: {},
+}));
+
+/// TODO: Padding is not showing at end of list (maybe storybook only?)
+
+interface HorizontalScrollerProps {
+  fetchNext?: () => void;
+  isFetching?: boolean;
+  className?: string;
+  children?: ReactNode;
+}
+
+export default function HorizontalScroller({
```

Comment:
This scroller is good for mobile only. The custom css scrollbar solution is crap (they don't disappear) and default mobile scrollbar looks good.
However I think horizontal scrollers are bad design on desktop anyway, so I think it's fine?